app/scripts/services/mobileClients.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+angular.module("openshiftConsole")
+  .factory("MobileClientsService", function(DataService) {
+
+    var mobileclientVersion = {
```

Comment:
would it be worth exposing this via the service?

Reply:
@maleck13 it is a service, having it as a factory is just a different way, which they seem to be using.